### PR TITLE
Upgrade RuboCop to 0.55.0.

### PR DESCRIPTION
**Pull request by dblock**
Replaces #72.

### CODE REVIEW

**Comment by rmm5t** [comment]
@dblock Sorry, I missed your comment on #76 about merging this one first. Could you please fix the merge conflicts here, and I'll try to get this merged in with a new official release soon after. Thanks!

**Comment by dblock** [comment]
Rebased, and corrected some obvious rubocop offenses.

File: mongoid-grid_fs.gemspec [hunk 1]
```diff
@@ -17,7 +17,6 @@ Gem::Specification.new do |spec|
   spec.test_files   = Dir['test/**/*']
   spec.require_path = 'lib'
 
-  spec.add_dependency 'mongoid',    '>= 3.0', '< 8.0'
   spec.add_dependency 'mime-types', '>= 1.0', '< 4.0'
-  spec.add_development_dependency 'minitest', '>= 5.7.0', '< 6.0'
```
[the remark this answers is not in the document]
Comment (dblock) [reply mongoid-grid_fs.gemspec]: For consistency with other gems. I think it should be one way or another.

**Comment by dblock** [comment]
@rmm5t Cool we merge this? I am happy to do a release next if it helps.

**Pull request merged**